pages/common/gh-release.md

```diff
@@ -9,20 +9,20 @@
 
 - Display information about a specific release:
 
-`gh release view {{tag}}`
+`gh release view "{{tag}}"`
 
 - Create a new release:
 
-`gh release create {{tag}}`
+`gh release create "{{tag}}"`
 
 - Delete a specific release:
 
-`gh release delete {{tag}}`
+`gh release delete "{{tag}}"`
 
 - Download assets from a specific release:
 
-`gh release download {{tag}}`
+`gh release download "{{tag}}"`
 
 - Upload assets to a specific release:
 
-`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}}`
+`gh release upload "{{tag}}" {{1 or more: path/to/file1 path/to/file2 ...}}`
```

Comment:
Same here.

Also, the `1 or more: ` would be confusing for beginners, since they would just type out those literal words and wonder why it didn't work. An alternative:
```suggestion
`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}} {{...}}`
```

tldr pages should be intuitive to read, not prescripted to a single style with technical syntax. The point of tldr-pages is to be easier to read and more intuitive than regular man pages. To this end, it is inevitable that tldr pages will all look  little different to one another. While fixing a few errors is good, ensuring all pages are rigorously adhering to a single style is not good, since each page has it's own circumstances which demand it be different to the rest..... even within the subcommands of a single command.

Reply:
`{{...}}` is incorrect syntax for ellipsis, the correct one is always just this: `{{placeholder1 placeholder2 ...}}`.

Reply:
I've removed `1 or more`.